lib/twilio/call.ts

```diff
@@ -851,6 +862,45 @@ class Call extends EventEmitter {
     }
   }
 
+  /**
+   * Send a message to Twilio.
+   *
+   * @example
+   * ```ts
+   * call.sendMessage(Call.MessageType.UserDefinedMessage, {
+   *   ahoy: 'world!,
+   * });
+   * ```
+   *
+   * @param messageType - The type of the message to send to Twilio.
+   * @param content - The content of the message to send to Twilio.
+   */
+  sendMessage(messageType: Call.MessageType, content: any) {
```

Comment:
Should this be a promise? Or is that a separate task? Right now it just a straight hand off to signaling.

Reply:
Yeah so that's the task that's rolling over, using the `ack` from signaling.